scripts/llnl/common_build_functions.py

```diff
@@ -288,19 +295,15 @@ def build_and_test_host_config(test_root, host_config, report_to_stdout=False, e
     print("[starting unit tests]")
     print("[log file: %s]" % tst_output_file)
 
-    tst_cmd = "cd %s && make CTEST_OUTPUT_ON_FAILURE=1 test ARGS=\"--no-compress-output -T Test -VV -j8\"" % build_dir
-
-    res = sexe(tst_cmd,
-               output_file = tst_output_file,
-               echo=True)
-
-    if report_to_stdout:
-        with open(tst_output_file, 'r') as test_out:
-            print(test_out.read())
+    tst_cmd = "cd %s && make CTEST_OUTPUT_ON_FAILURE=1 test ARGS=\"--no-compress-output -T Test -VV -j %s\"" % (build_dir, job_count)
```

Comment:
This can't go this high due to the parallelism we are using at the test level with MPI and OpenMP.

I think the formula is actually `(total CPU cores) / (highest level of OpenMP or MPI per test [pretty sure it's 4]) = allowable job slots on the test line`

8 has never caused a problem for us because usually people have 32 cores.

Reply:
How would you suggest handling this?

I could set it to 8 if `jobs` is an empty string. Or I can create a second `jobs-test` argument?

Reply:
I would either leave it at 8 or honor the option if it is less than 8.